<launch>
    <node name="fetcher" pkg="tron_future" type="fetchRadar_node" output="screen"/>
</launch>